docs/source/api/apollo-server.md

```diff
@@ -342,20 +342,57 @@ addMockFunctionsToSchema({
    By default, errors sending reports to Engine servers will be logged
    to standard error. Specify this function to process errors in a different
    way.
+   
+* `sendVariableValues`: { transform: (options: { variables: Record<string, any>, operationString?: string } ) => Record<string, any> }
+                     | { exceptNames: Array&lt;String&gt; }
+                     | { includeNames: Array&lt;String&gt; }
+                     | { sendNone: true }
+                     | { sendAll: true }
+    
+    By default, Apollo Server does not send the values of any GraphQL variables to Apollo's servers, because variable values often contain the private data of your app's users. If you'd like variable values to be included in traces, set this option. This option can take several forms:
+    
+    - { sendNone: true }: blocklist all variable values (DEFAULT)
```

Comment:
I don't think we need special vocab here. "don't send any variable values", "send all variable values"